TriblerGUI/qt_resources/mainwindow.ui

```diff
@@ -3307,7 +3310,7 @@ font-weight: bold;</string>
               </widget>
              </item>
              <item>
-              <widget class="QLabel" name="num_search_results_label">
+              <widget class="QLabel" name="num_results_label">
```

Comment:
In Qt, names of `QWidget` objects should be unique across the whole `.ui` file. That's why I name these objects as specific as possible. The name `num_results_label` could result in a name collision.

Reply:
Hmm. There are not collisions now. I guess we're fine with this name now. We can always rename it later.